gclient/src/api/error.rs

```diff
@@ -89,4 +89,10 @@ pub enum Error {
     /// Occurs when node spawining failed.
     #[error(transparent)]
     Node(#[from] NodeError),
+    /// Occurs when parsing websocket domain failed.
+    #[error("Failed to parse WebSocket domain.")]
+    IncorrectWSDomain,
```

Comment:
Should be somthing like `WSDomainIncorrect` so it is consistent with errors like `TimestampNotFound`, `MaxDepthReached`